index.html

```diff
@@ -33,5 +33,166 @@ <h1 class="top-name">ELDON RATZLAFF</h1>
 <div class="ornamen-first-botom">
 </div>
 </section>
+
+
+<div class="projects">Projects</div>
+
+<section class="midle-section">
+<!-- proyect 1 -->
+    <div class="project1"></div>
+    <div class="proyect-foot1">
+        <p class="proyect-foot1-p">Project name goes here</p>
+        <ul class="proyect-foot1-ul">
+            <li class="proyect-foot1-li">HTML/CSS</li>
+            <li class="proyect-foot1-li">Ruby on Rails</li>
+            <li class="proyect-foot1-li">JavaScript</li>
+        </ul>
+
+        <div class="proyect-foot1-button">
+            <h3 class="proyect-foot1-text">See this project</h3>
+            <div class="proyect-foot1-img"></div>
+        </div>
+    </div  >     
```

Comment:
- [ ] Please, kindly consider wrapping each of the `project-card` in a single `tag`, (e.g `<div>`) then align the content of each `project-card` with Flexbox as mentioned in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)

Your project card is not wrapped together 👇 
```
<!-- proyect 1 -->

    <div class="project1"></div>
    <div class="proyect-foot1">
        . . . 
    </div  >     
```
- [ ] Let us make the project title use a header tag `hx`, that way it will be semantically correct 👍. 👇 

```
<p class="proyect-foot1-p">Project name goes here</p>
```
![image](https://user-images.githubusercontent.com/78251927/205418225-1125b964-5b73-4ee4-bc4d-33ac748e6e84.png)

- [ ] Please kindly make the "See this project" a link `a` or `button` tag, 👇 for semantic reasons.
```
. . . 
<h3 class="proyect-foot1-text">See this project</h3>
 . . .
```

![image](https://user-images.githubusercontent.com/78251927/205418096-b3d7460a-c5d6-44bf-b0de-2e3b67a31c77.png)